blobs.go

```diff
@@ -185,6 +190,16 @@ type BlobWriter interface {
 	Reader() (io.ReadCloser, error)
 }
 
+// BlobKeeper combines the operations needed for registry's upkeeping such as
+// blob access, read, enumeration and delete. Can be used to prune unnecessary
+// objects.
+type BlobKeeper interface {
```

Comment:
`BlobStore` already contains `BlobDeleter` and `BlobService` which in turn contains `BlobStatter` and `BlobProvider`.